.github/workflows/containerised-test.yml

```
@@ -22,7 +22,26 @@ on:
         type: boolean
         default: false
 
+env:
+  CRYSTAL_VERSION: 1.4.1
+
 jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run docker-compose test environment
+        run: ./test --coverage
```

I think it's worth keeping randomised spec order.
That shouldn't affect the branch coverage.

```suggestion
        run: ./test --coverage --order=random
```

---

```sh
/app # crystal-coverage -p | crystal eval --order=random
syntax error in eval:1
Error: postfix decrement is not supported, use `exp -= 1`
```